src/Tasks.UnitTests/AssemblyDependency/Miscellaneous.cs

```diff
@@ -7742,12 +7742,12 @@ public void Regress397129_HandleInvalidDirectoriesAndFiles_Case2()
         /// Consider this dependency chain:
         ///
         /// App
-        ///   References - A
-        ///        Depends on B
-        ///        Will be found by hintpath.
-        ///   References -B
-        ///        No hintpath
-        ///        Exists in A.dll's folder.
+        /// References - A
+        /// Depends on B
+        /// Will be found by hintpath.
+        /// References -B
+        /// No hintpath
+        /// Exists in A.dll's folder.
```

Comment:
⚠️ This change negatively affects readability. My suggestion is to wrap the paragraph in `<code>`/`</code>` to preserve the original formatting.